trio_run_in_process/abc.py

```diff
@@ -120,3 +121,15 @@ def terminate(self) -> None:
     @abstractmethod
     def send_signal(self, sig: int) -> None:
         ...
+
+
+class WorkerProcessAPI(ABC):
+    @abstractmethod
+    async def open(
```

Comment:
You need to drop the `async` here and mypy will be happy.

See: https://github.com/ethereum/ddht/pull/261#discussion_r539417748